pulp_container/app/registry_api.py

```diff
@@ -951,13 +993,30 @@ def handle_safe_method(self, request, path, pk):
             try:
                 tag = models.Tag.objects.get(name=pk, pk__in=repository_version.content)
```

Comment:
somewhere we need to add a check: if tag exists locally is it up-to-date with the remote source? ( issue a head request)

Reply:
I will move this check to content-app since that one would deal with content that should be downloaded. I find it redundant to fetch a manifest from api-app because we cannot share the information about updated content with content-app.